apps/src/code-studio/components/progress/BulkLessonVisibilityToggle.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {getStore} from '../../../redux';
+import PropTypes from 'prop-types';
+import Button from '@cdo/apps/templates/Button';
+import FontAwesome from '../../../templates/FontAwesome';
+import ReactTooltip from 'react-tooltip';
+import {toggleHiddenLesson} from '@cdo/apps/code-studio/hiddenLessonRedux';
+import {unitCalendarLesson} from '../../../templates/progress/unitCalendarLessonShapes';
+import _ from 'lodash';
+import style from './bulk-lesson-visibility-toggle.module.scss';
+import i18n from '@cdo/locale';
+
+function BulkLessonVisibilityToggle({lessons, sectionId, scriptName}) {
```

Comment:
if it's not too much trouble, it would be great to use `unitName` instead of `scriptName`. We're in the process of change the term "script" to "unit"